variables.tf

```diff
@@ -387,6 +387,18 @@ variable "fargate_profiles" {
   default     = {}
 }
 
+variable "fargate_profiles_create_timeout" {
```

Comment:
We dont want add additionall variables into submodule, it should follow previously used pattern. I dont know why you changed this as I given you direct suggestion how to make default null value